### ISSUE TITLE
Function list for the common global scripts

### ISSUE THREAD

**Issue by kinzlaw** [immense/immybot-documentation#10]
Hi, there!
I suggest maintaining a function list for the common global scripts(in a well-grouped manner), such that users can better utilize them, and prevent users from reinventing the wheel. Thanks.

**Comment by kinzlaw** [comment]
As an example, I see that there are multiple 'EXE generic install *' scattered around, which is quite informal, and tends to mislead the users to create their custom scripts.

**Comment by dkattan** [comment]
I've avoided documenting them because they are a mess.

I just cleaned up all scripts with EXE in the name. The "Generic EXE Install" was actually for "Visual Studio" installers used for .NET runtimes and VS Related Tooling

**Comment by kinzlaw** [comment]
Yes, cleaning the existing functions up and making them is surely time-consuming, but it's better to do it earlier than later.
